web/test/components/normUnitList.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from "@testing-library/vue"
+import NormsList from "@/components/NormsList.vue"
+import Norm from "@/domain/Norm"
+
+describe("norms list", () => {
+  test("renders fallback text if no norms are found", async () => {
+    render(NormsList, {
+      props: {
+        norms: [],
+      },
+    })
+
+    await screen.findByText("Keine Dokumentationseinheiten gefunden")
+  })
+
+  test("renders list of norms", async () => {
+    const longTitle = "test"
+    const guid = "123"
+    const articleMock = [
```

Comment:
No critics at all, rather a knowledge sharing:
We use the concept of data generator functions for domain objects. If you wanna get inspired checkout [this file](https://github.com/digitalservicebund/ris-backend-service/blob/main/web/test/test-helper/dataGenerators.ts). Looking forward to see some norms related generators soon.
For some context: generator functions allow shorter tests that are easier to read and only express the parts that are actually relevant for this specific test case.